SzczepionkaAPI/src/main/java/com/szczepionka/configuration/ApplicationConfig.java

```diff
@@ -31,4 +32,9 @@ public DistanceCalculator distanceCalculator() {
     public ZipcodeConverter zipcodeConverter() {
         return new ZipcodeConverter();
     }
+
+    @Bean
+    public PatientLinkGenerator patientLinkGenerator() {
+        return new PatientLinkGenerator();
```

Comment:
this is a static class, why have you added it as a @Bean? Is it neccessary?

Reply:
fixed